script.js

```diff
@@ -0,0 +1,26 @@
+	var immigration = document.getElementById("immigrationBlurb");
+	var btn = document.getElementById("btn");
+
+	btn.addEventListener("click", function() {
+		var ourRequest = new XMLHttpRequest();
+		ourRequest.open('GET', 'https://code-your-future.github.io/api-demo/area/index.json', true);
+		
+		ourRequest.onload = function () {
+		var ourData = JSON.parse(ourRequest.responseText);

```

Comment:
I would probably just pass `ourRequest.responseText` into `renderHTML()` and to the `JSON.parse` there so this function doesn't do more than that.

It works either way, but by reducing the complexity of your XHR request to only handling the GET and passing the data off will only be a good thing in the long run.